### DISCUSSION TITLE
work package 📦

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/progettazione-di-algoritmi#66] [Esercizi A-L]
Dato un progetto diviso in $x_1, x_2, ..., x_n$ work package, date rispettive dipendenze (Es. $x_1$ va svolto dopo $x_{12}$) e dato il tempo necessario al completamento di ogni work package, scrivere lo pseudocodice di un algoritmo che indica il tempo in cui deve essere iniziato ogni work package, in modo da rispettare le dipendenze.

**Comment by cek753** [comment]
<img width="1647" height="1389" alt="image" src="https://github.com/user-attachments/assets/0eb1136d-d33b-4098-b77f-22a6bff5a17d" />
